src/roaring.c

```diff
@@ -2005,20 +2006,26 @@ static void inplace_fully_flip_container(roaring_array_t *x1_arr, uint16_t hb) {
 roaring_bitmap_t *roaring_bitmap_flip(const roaring_bitmap_t *x1,
                                       uint64_t range_start,
                                       uint64_t range_end) {
-    if (range_start >= range_end) {
+    if (range_start >= range_end || range_start > (uint64_t)UINT_MAX + 1)
```

Comment:
```suggestion
    if (range_start >= range_end || range_start > (uint64_t)UINT32_MAX + 1)
```

In theory, UINT_MAX could be UINT64_MAX (I don't think this actually happens, but it might).